src/imitation/util/sacred.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2019 Google LLC.
```

Comment:
Unclear if Google really deserves copyright here. Your code looks very different from my Gist. Question of whether you just read it for inspiration, or iteratively edited it to reach this point.

If we think Google does deserve copyright here, then should put the full Apache 2.0 license header here. You should also add your name: it's not just a Google work.

Otherwise, I think it can be MIT licensed like the rest of this codebase.

Reply:
Seems mostly like inspiration at this point. The "check that no filtered directory is a prefix of a filtered directory" was copied directly from your code however.

Reply:
Removing copyright